src/engraving/libmscore/chord.cpp

```diff
@@ -848,7 +848,7 @@ void Chord::addLedgerLines()
     }
 
     // the extra length of a ledger line to be added on each side of the notehead
-    double extraLen = score()->styleMM(Sid::ledgerLineLength) * mag;
+    double extraLen = score()->styleMM(Sid::ledgerLineLength); // no magnitude here, as individual note size will be taken into count
```

Comment:
I added comment, as one could expect magnitude there :) OK. Done

Reply:
@sammik I understand, but let's not add comments for every small detail, only for very peculiar things. Otherwise we end up with more comments than code :)